deploy/application/files/values.yaml.tpl

```diff
@@ -21,18 +21,27 @@ env:
   ${key}: ${value}
 %{ endfor ~}
 
+{%- if app_url_type == "path" %}
+targetGroupBinding:
+  enabled: ${enable_target_group_binding}
+  targetGroupARN: ${target_group_arn}
+{%- endif %}
+
+{%- if app_url_type == "subdomain" %}
 ingress:
   className: alb
   annotations:
-    alb.ingress.kubernetes.io/listen-ports: '[{"HTTPS":443}]'
-    alb.ingress.kubernetes.io/scheme: internet-facing
-    alb.ingress.kubernetes.io/ssl-policy: ELBSecurityPolicy-TLS13-1-2-2021-06
```

Comment:
@DCamma Was there a reason to remove the `ssl-policy`? Is there a good default from AWS?